java/src/jmri/jmrit/logixng/tools/swing/TreeViewer.java

```diff
@@ -76,7 +76,7 @@ public void setRootVisible(boolean rootVisible) {
 
     @Override
     public void dispose() {
-        _treePane.dispose();
+        // _treePane.dispose();
         super.dispose();
     }
```

Comment:
Thanks for your work on this!

GUI coding is not my strong side. But this class `TreeViewer` extends `JmriJFrame`. And TreeViewer creates a `TreePane` which extends `JPanel`. That TreePane is then added to the content pane of this TreeViewer. When I think about it, I guess that the JFrame will dispose the JPanel, meaning that this entire `dispose()` method can be removed.

https://github.com/JMRI/JMRI/blob/master/java/src/jmri/jmrit/logixng/tools/swing/TreeViewer.java
https://github.com/JMRI/JMRI/blob/master/java/src/jmri/jmrit/logixng/tools/swing/TreePane.java